Extend/Model/Order.php

```diff
@@ -53,18 +54,38 @@ public function isModulePaymentType(): bool
     }
 
     /**
-     * Returns an array of available states.
+     * Returns an associative array of available states and their translation.
      *
      * @return array
      */
-    public static function getStates(): array
+    public static function getTranslatedStates(): array
```

Comment:
Can be private if you don't plan on using it elsewhere.

Reply:
no it is also used in OrderList